client/dashboard/data/domains.ts

```diff
@@ -6,7 +6,6 @@ export type { DomainSuggestion, DomainSuggestionQuery };
 
 export const DomainTypes = {
 	MAPPED: 'mapping',
-	REGISTERED: 'registered',
```

Comment:
This one is computed domain type by `! wpcom_domain && has_registration`. Try to get rid of it and use the raw data to determine whether the domain is registered.

Reply:
The registered value is not returned by the backend; it is computed locally using `createSiteDomainObject` and `getDomainType`.